stubs/pyjks/jks/__init__.pyi

```diff
@@ -0,0 +1,52 @@
+from .bks import *
+from .jks import *
+from .jks import __version__, __version_info__
+from .util import *
+
+# pyjks exports lots of junk such as jks.jks.SIGNATURE_WHITENING, jks.util.b8 etc.
+# We don't mark those as re-exported as those don't seem like intended part of the public API.
```

Comment:
Does this make sense? (in which case these need allowlist entries)

Or should I just blindly re-export everything that `pyjks` does and not worry about it?

Reply:
It's not the approach we usually take, since this implies that somebody can do something like this, which will obviously fail at runtime:

```pycon
>>> import jks
>>> for attr in jks.__all__:
...     do_thing_with_attr(attr)
```

The approach we usually take is to explicitly re-export the things that are meant to be re-exported (`from jks import foo as foo, bar as bar`, etc.) and, if necessary, allowlist all the things that are just incidentally imported and aren't really meant to be publicly reexported.